CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# dbt_salesforce_formula_utils v0.9.2
+##  Documentation Update
+- Fivetran will be deprecating support for Options 2 and 3 by October 1 where you can set `full_statement_version=false`. We've highlighted this change to users of the package [in the README](https://github.com/fivetran/dbt_salesforce_formula_utils#step-4-create-models).
```

Comment:
Users may not be entirely aware of what Options 2 and 3 are. Let's just highlight that we are deprecating the config. I feel that would resonate more.

```suggestion
- Fivetran will be deprecating support for the `full_statement_version=false` config by October 1, 2023. We've highlighted this change to users of the package [in the README](https://github.com/fivetran/dbt_salesforce_formula_utils#step-4-create-models).
```

Reply:
Updated!